deps/rabbitmq_prometheus/Makefile

```diff
@@ -26,6 +26,12 @@ endif
 include ../../rabbitmq-components.mk
 include ../../erlang.mk
 
+# We are using stream_test_utils from rabbitmq_stream.
+CT_OPTS += -pa ../rabbitmq_stream/test/
+
+test-build::
+	$(verbose) $(MAKE) -C ../rabbitmq_stream test-dir
```

Comment:
Don't do this unless there's no other choice. In this case it's probably better to move the module to rabbitmq_ct_helpers.